### PR TITLE
Fix seeing surgeries while blind

### LINKED ISSUE #53295: While asleep, you can see people doing surgery on you.
## Reproduction:
Go to sleep, have someone start doing surgery.

Possible oversight?
image

### PR DESCRIPTION
**Pull request by Mothblocks**
## About The Pull Request
Fixes #53295.

## Changelog
:cl:
fix: You can no longer see what surgery steps are being performed on you while blind.
/:cl:

### CODE REVIEW

**Review by Rohesie — DISMISSED** [review]

**Comment by msgerbs** [comment]
Idk about you but I feel like if I was blind and awake I'd feel somebody cutting my skull open, ya know? I agree it shouldn't be shown for unconscious people but blind people should definitely know what's being done to their body if they're awake/

**Comment by msgerbs** [comment]
Also we currently have the exact opposite issue open? #51219

**Comment by MrMelbert** [comment]
> 
> 
> Also we currently have the exact opposite issue open? #51219

this is a different issue entirely - your linked issue refers to messages displayed to blind people doing surgery itself

as for this pr - i'm also pretty sure awake and blind people would be able to feel people doing surgery on them. this would leave blind people completely guessing whether or not someone is AFK or actually doing stuff to them in surgery which is kinda weird

File: code/modules/surgery/surgery_step.dm [hunk 1]
```diff
@@ -175,4 +175,4 @@
 /datum/surgery_step/proc/display_results(mob/user, mob/living/carbon/target, self_message, detailed_message, vague_message, target_detailed = FALSE)
 	user.visible_message(detailed_message, self_message, vision_distance = 1, ignored_mobs = target_detailed ? null : target)
 	if(!target_detailed)
-		to_chat(target, vague_message)
+		target.show_message(vague_message, MSG_VISUAL)
```
Comment (Timberpoes) [inline code/modules/surgery/surgery_step.dm:178]: Mentioned it on Discord but I'll double it up here.

Can ya add a cheeky little alt_msg to show_message, something generic that lets blind people know that some surgery procedure is happening to them and it is causing them pain.

Once that is done, this is all good to merge in my opinion!

**Review by Timberpoes — CHANGES_REQUESTED** [review]

File: code/modules/surgery/surgery_step.dm (hunk 1 above)
Comment (Jared-Fogle) [reply code/modules/surgery/surgery_step.dm]: Done.

**Review by Timberpoes — APPROVED** [review]

**Pull request merged**